### PR TITLE
[FIX] #427 and manage suggestion selected

### PR DESCRIPTION
**Pull request by zarnifoulette**

### CODE REVIEW

File: src/main/resources/public/js/pages/store.js [hunk 1]
```diff
@@ -179,8 +179,8 @@ const SearchAppsForm = createClass({
     fullTextSearchChanged: function (event) {
         this.props.updateFilter(null, "searchText", event.target.value);
     },
-    onGeoChange: function (geoArea) {
-        this.props.updateFilter(null, "geoAreaAncestorsUris", geoArea.ancestors);
+    onGeoChange: function (event, newValue) {
+        this.props.updateFilter(null, "geoAreaAncestorsUris", newValue);
```
Comment (bobeal) [inline src/main/resources/public/js/pages/store.js]: not sure about this ancestors thing. I have to check how it is supposed to be used

**Comment by bobeal** [comment]
There are some issues in the store's form :
* A new search is performed each time a new character is entered. But no search should be performed until a suggestion from the list is selected (because they are matched against URIs, not fulltext)
* After selecting a suggestion, two searches are performed : one with the selected geo area (this is good) and another one just after with no geo area
* The kernel is expecting an URI, not a free string (thus, I think, the previous geoancestors_uri field)

**Comment by bobeal** [comment]
PS : even when fixing a bug, prefix your branch by `feature`, not `bug` :)

**Comment by zarnifoulette** [comment]
That is fixed. I tried to rename the branch (without success) but that just delete this one and create a new one, so I will make a new PR.

**Pull request closed without merging**